PIPs/pip-37.md

```diff
@@ -47,6 +47,27 @@ altering the existing structure for single-payload transactions.
 Transactions using the old format remain valid,
 while the new format can be extended to support batch transactions.
 
```

Comment:
I think this phrase is fair enough: `The version for transactions with multiple payloads should be set to 2.`